From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Sun, 12 Jul 2020 13:39:32 +0100
Subject: [PATCH] Revert "dt-bindings: Add doc for Pine64 Pinebook Pro"

This reverts commit ec0f66d2bc5ffd2a06eaf254ad314153e557cb7c.

This is already upstream and not sure why it still applies but we
don't need it any more.
---
 Documentation/devicetree/bindings/arm/rockchip.yaml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Documentation/devicetree/bindings/arm/rockchip.yaml b/Documentation/devicetree/bindings/arm/rockchip.yaml
index ed808148b6a7..d4a4045092df 100644
--- a/Documentation/devicetree/bindings/arm/rockchip.yaml
+++ b/Documentation/devicetree/bindings/arm/rockchip.yaml
@@ -425,11 +425,6 @@ properties:
           - const: pine64,rockpro64
           - const: rockchip,rk3399

-      - description: Pine64 PinebookPro
-        items:
-          - const: pine64,pinebook-pro
-          - const: rockchip,rk3399
-
       - description: Radxa Rock
         items:
           - const: radxa,rock
-- 
2.26.2

